### PR TITLE
Spelling misc

### PR DESCRIPTION
**Pull request by jsoref**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/ImageMagick/ImageMagick/pulls) open
- [x] I have verified that I am following the existing coding patterns and practices as demonstrated in the repository.

### Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

Note: I'll try to annotate the changes. For medium size PRs, I tend to be asked to split them -- I can do this according to project requests -- my general preference is by directory/file type -- it's very expensive for me to split by "is comment" vs "is api" vs "is function", although it is doable (it just takes longer and is generally more fragile as I tend to squash and then select which means that later adjustments are much harder). In general, I prefer to drop problematic changes first (before squashing).

The misspellings have been reported at https://github.com/jsoref/ImageMagick/commit/b59c962bd0e91d1533fcc0b089ab75869abcfe3f#commitcomment-90677150

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/ImageMagick/commit/066cbc34c59a678049650598a950a5ec1e959bb5

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: www/cite.html [hunk 1]
```diff
@@ -101,7 +101,7 @@ <h4>Citation in Vancouver style</h4>
 <pre class="bg-light text-dark mx-4"><samp>1. The ImageMagick Development Team. ImageMagick [Internet]. 2021. Available from: https://imagemagick.org</samp></pre>
 
 <h4>Citation in Harvard style</h4>
-<pre class="bg-light text-dark mx-4"><sampl>The ImageMagick Development Team, 2021. ImageMagick, Available at: https://imagemagick.org.</samp></pre>
+<pre class="bg-light text-dark mx-4"><samp>The ImageMagick Development Team, 2021. ImageMagick, Available at: https://imagemagick.org.</samp></pre>
```
Comment (jsoref) [inline www/cite.html:104]: notable

File: www/conjure.html [hunk 2]
```diff
@@ -566,7 +566,7 @@ <h2><a class="anchor" id="msl"></a>Magick Scripting Language</h2>
 
   <tr>
     <td><strike>evaluate</strike></td>
-    <td>value="double", operator=""Add, And, Divide, LeftShift, Max, Min, Multiply, Or, Rightshift, Subtract, Xor"", channel="All, Default, Alpha, Black, Blue, CMYK, Cyan, Gray, Green, Index, Magenta, Opacity, Red, RGB, Yellow" </td>
+    <td>value="double", operator="Add, And, Divide, LeftShift, Max, Min, Multiply, Or, Rightshift, Subtract, Xor", channel="All, Default, Alpha, Black, Blue, CMYK, Cyan, Gray, Green, Index, Magenta, Opacity, Red, RGB, Yellow" </td>
```
Comment (jsoref) [inline www/conjure.html:569]: this isn't a spelling fix... it just looked odd...

File: www/source/english.xml [hunk 3]
```diff
@@ -610,7 +610,7 @@
         <message name="CompositeImageRequired">
           composite image required
         </message>
-        <message name="CurlyBrasesNestedTooDeeply">
+        <message name="CurlyBracesNestedTooDeeply">
```
Comment (jsoref) [inline www/source/english.xml:613]: notable. these changes presumably break localizations -- each project has a different policy about localizations, it's a toss up between "please update" and "please don't touch"...

File: www/support.html [hunk 4]
```diff
@@ -136,13 +136,13 @@ <h2><a class="anchor" id="support"></a>Sponsor the ImageMagick Project</h2>
...
-<h2>Github</a></h2>
+<h2>GitHub</a></h2>
 <p>ImageMagick Studio LLC is the organization that designs, maintains, and enhances the ImageMagick software suite. In the past, we paid for the privilege of producing free software out of our own pockets. Now, to help defray costs associated with network bandwidth, development software, computer hardware, taxes, legal, and other expenses, we accept donations from individuals or companies like yourself.</p>
 <ul>
 <iframe src="https://github.com/sponsors/ImageMagick/button" title="Sponsor ImageMagick" height="35" width="107" style="border: 0;"></iframe>
 </ul>
-<h2><a href="https://www.paypal.com/biz/fund?id=9XPKRUWDXRR8J">Paypal</a></h2>
-<p>For direct monetary contributions, click on the donate button to contribute thru Paypal or with a credit card:</p>
+<h2><a href="https://www.paypal.com/biz/fund?id=9XPKRUWDXRR8J">PayPal</a></h2>
+<p>For direct monetary contributions, click on the donate button to contribute thru PayPal or with a credit card:</p>
```
Comment (jsoref) [inline www/support.html:145]: brands

File: Magick++/lib/Magick++/Image.h [hunk 5]
```diff
@@ -676,7 +676,7 @@ namespace Magick
 
     // Changes the brightness and/or contrast of an image. It converts the
     // brightness and contrast parameters into slope and intercept and calls
-    // a polynomical function to apply to the image.
+    // a polynomial function to apply to the image.
```
[the remark this answers is not in the document]
Comment (dlemstra) [reply Magick++/lib/Magick++/Image.h:679]: @fmw42 Can you confirm?

File: MagickCore/cache.c [hunk 6]
```diff
@@ -4278,7 +4278,7 @@ static Quantum *QueueAuthenticPixelsCache(Image *image,const ssize_t x,
 %  Write-only access means that any existing pixel values corresponding to
 %  the region are ignored.  This is useful if the initial image is being
 %  created from scratch, or if the existing pixel values are to be
-%  completely replaced without need to refer to their pre-existing values.
+%  completely replaced without need to refer to their preexisting values.
```
Comment (dlemstra) [inline MagickCore/cache.c:4281]: @fmw42 Can you confirm this change? pre-existing looks okay to me?

File: MagickCore/display.c [hunk 7]
```diff
@@ -1391,7 +1391,7 @@ typedef enum
   AddFrameCommand,
   CommentCommand,
   LaunchCommand,
-  RegionofInterestCommand,
+  RegionOfInterestCommand,
```
[the remark this answers is not in the document]
Comment (dlemstra) [reply MagickCore/display.c:1394]: Don't think we can make this change. Can you confirm @urban-warrior?

File: MagickCore/distort.c [hunk 8]
```diff
@@ -451,7 +451,7 @@ static double *GenerateCoefficients(const Image *image,
       cp_values = 2;
       break;
 #if 0
-    case QuadraterialDistortion:
+    case QuadrilateralDistortion:
```
[the remark this answers is not in the document]
Comment (dlemstra) [reply MagickCore/distort.c:454]: Appears not to be used so safe to change.

**Review by dlemstra — CHANGES_REQUESTED** [review]
It would be a lot easier to review and merge this changes if it would be split up in smaller PR's. Can you please create a separate pull request for the following folders?

- Magick++
- MagickCore
- MagickWand
- PerlMagick
- coders
- www
- rest/remaining

File: MagickCore/display.c (hunk 7 above)
[the remark this answers is not in the document]
Comment (urban-warrior) [reply MagickCore/display.c:1394]: An ENUM that is not in a public header, can be changed.  So changing RegionofInterestCommand is fine.

**Comment by jsoref** [comment]
Sure. Hopefully tonight.

**Review by dlemstra — APPROVED** [review]

**Pull request merged**